.github/workflows/ci.yml

```
@@ -110,6 +112,11 @@ jobs:
         if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
       - name: Build latest earthly using released earthly
         run: earthly --use-inline-cache +for-linux
+      - name: Set EARTHLY_VERSION_FLAG_OVERRIDES env
+        run: |-
+            set -euo pipefail
+            EARTHLY_VERSION_FLAG_OVERRIDES="$(tr -d '\n' < .earthly_version_flag_overrides)"
+            echo "EARTHLY_VERSION_FLAG_OVERRIDES=$EARTHLY_VERSION_FLAG_OVERRIDES" >> "$GITHUB_ENV"
```

Would this work on a single line?

```suggestion
            echo "EARTHLY_VERSION_FLAG_OVERRIDES=$(tr -d '\n' < .earthly_version_flag_overrides)" >> "$GITHUB_ENV"
```

---

similarly I had to break this up to address error checking